apps/wallet/src/ui/app/pages/home/apps/index.tsx

```diff
@@ -1,31 +1,54 @@
 // Copyright (c) Mysten Labs, Inc.
 // SPDX-License-Identifier: Apache-2.0
 
+import { useFeature } from '@growthbook/growthbook-react';
 import { Route, Routes } from 'react-router-dom';
 
 import { Content } from '_app/shared/bottom-menu-layout';
 import FiltersPortal from '_components/filters-tags';
 import AppsPlayGround, { ConnectedAppsCard } from '_components/sui-apps';
+import { FEATURES } from '_src/shared/experimentation/features';
+
+import type { DAppEntry } from '_src/ui/app/components/sui-apps/SuiApp';
 
 import st from './AppsPage.module.scss';
 
+type FilterTag = {
+    name: string;
+    link: string;
+};
+
 function AppsPage() {
-    const filterTags = [
+    const defaultFilterTags: FilterTag[] = [
         {
-            name: 'Playground',
-            link: 'apps',
+            name: 'Connections',
+            link: 'apps/connected',
         },
         {
-            name: 'Active Connections',
-            link: 'apps/connected',
+            name: 'All',
+            link: 'apps',
         },
     ];
+    const ecosystemApps =
+        useFeature<DAppEntry[]>(FEATURES.WALLET_DAPPS).value ?? [];
+    const uniqueAppTags = new Set<FilterTag>();
+
+    ecosystemApps.forEach((app) => {
+        app.tags.forEach((tag) => {
+            uniqueAppTags.add({
```

Comment:
I am not sure that this will produce unique tags. I would use the `tag` for the `Set` and then `map` it to the final object